### ISSUE TITLE
Not an issue, but a question. Ending a pyttsx thread

### ISSUE THREAD

**Issue by gyaaniguy** [parente/pyttsx#4]
in my python program I am starting a thread. This thread creates a engine object and invokes engine.runAndWait() . engine.stop() (inside onWord()) is called to stop the engine and end the thread. After this another identical thread using the same function is created . Now the problem starts. even though the first thread is dead. something still interferes with the second thread and does not allow it to work properly.

Basically what I want to do is stop a previous pyttsx instance and start another pyttsx instance. All this effort is  directed at having something like a pause feature...

Here is the full code of my tiny 'program' : http://pastebin.com/dnCd3KjS

Hope I was able to make myself clear.
